src/Telegram.php

```diff
@@ -127,14 +126,15 @@ class Telegram
      *
      * @var array
      */
-
-    protected $message_types =['Message', 'Photo', 'Audio', 'Document', 'Sticker', 'Video',
+    protected $message_types = [
+        'Message', 'Photo', 'Audio', 'Document', 'Sticker', 'Video',
         'Voice', 'Location', 'command', 'new_chat_participant',
         'left_chat_participant', 'new_chat_title', 'delete_chat_photo',
         'group_chat_created', 'supergroup_chat_created', 'channel_chat_created',
```

Comment:
This array's last item ends with a comma. Other arrays don't.
Is this intentional or a forgotten comma?
Personally I prefer having a leading comma, as it prevents errors when reordering array code for example.